### PR TITLE
libimobiledevice suite update, New packages: libirecovery and idevicerestore

### PR DESCRIPTION
**Pull request by CRKatri**

### CODE REVIEW

File: srcpkgs/libimobiledevice/template [hunk 1]
```diff
@@ -11,17 +11,14 @@ maintainer="Orphaned <orphan@voidlinux.org>"
...
-checksum=8ba1ea817691b3548b83371f8ec22d73d1e37f8fda7c899ac9c7121d924ca830
+checksum=acbfb73eabee162e64c0d9de207d71c0a5f47c40cd5ad32a5097f734328ce10a
 
 if [ "$CROSS_BUILD" ]; then
 	configure_args="ac_cv_sys_file_offset_bits=yes ac_cv_sys_large_file=yes"
 fi
 
 pre_configure() {
-	sed -i 's,SSLv3_,SSLv23_,g' src/idevice.c
 	sed -i 's,-L$(libdir),,g' cython/Makefile.am
```
Comment (ericonr) [inline srcpkgs/libimobiledevice/template:21]: please use vsed here

File: srcpkgs/libplist/template [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'libplist'
 pkgname=libplist
-version=2.1.0
+version=2.2.0
```
Comment (ericonr) [inline srcpkgs/libplist/template:3]: I guess changing python version didn't work? :/

Comment (CRKatri) [reply srcpkgs/libplist/template:3]: Slipped my mind, fixed now.

**Comment by ndowens** [comment]
Is this still being worked on?

**Comment by ahesford** [comment]
This was missing some revbumps for the `libplist` update, but most of this has been superseded by 92a1dc2acdb1a951841016a7b5138808c20807d0...a9b46a6be51245a771f621917c63c9bfa327e59b anyway. If there is still value in the new packages, the conflicting bumps will need to be dropped.

**Comment by ericonr** [comment]
@CRKatri do you have anything to add on top of the changes shown by @ahesford ?

**Pull request closed without merging**